README.md

```diff
@@ -34,20 +34,23 @@ See: https://b3n.org/automatic-ripping-machine
 
 ## Install
 
+    sudo apt-get install git
+    sudo apt-get install regionset
+    sudo regionset /dev/sr0
```

Comment:
So I have fixed everything and made it even better for customizing the file permissions based on your suggestion. I am testing it now but very confident in it. For the README.md, I'll let you handle updating that, you can just revert mine and use my suggestions to help update the original.